docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+
+services:
+
+  postgres:
+    image: postgres:13
+    environment:
+      POSTGRES_DB: aurora_store
+      POSTGRES_USER: ilya
+      POSTGRES_PASSWORD: 111111
```

Comment:
FYI: For testing purposes ok, but you can hide sensitive data into environment variables, locally by creating the `.env` file and putting them there (without committing the `.env` file), and in GitHub project setting environment variables as separate setting, with the same naming, which can be seen only by project owner (you).